bin/make-consensus.py

```diff
@@ -84,6 +84,16 @@ def main():
               'https://github.com/broadinstitute/picard for details on '
               'Picard.'))
 
+    parser.add_argument(
+        '--ivar', default=False, action='store_true',
+        help=('If given, ivar will be used to call the consensus.'))
+
+    parser.add_argument(
+        '--ivarFrequencyThreshold', default='0.6',
```

Comment:
Why not use `type=float` and `default=0.6`?  Then `argparse` can detect errors for you.

Reply:
Done.